app/models/offence.rb

```diff
@@ -28,4 +34,9 @@ class Offence < ActiveRecord::Base
   def offence_class_description
     offence_class.letter_and_description
   end
+
+  def offence_class_xor_fee_band
+    return if offence_class.present? ^ fee_band.present?
+    errors[:base] << 'Specify an Offence or a FeeBand, not both'
```

Comment:
**[not a blocker]** 💄   Consider moving the error message to the locales.

Reply:
i agree